### PR TITLE
Create additional-resources page

### PR DESCRIPTION
**Pull request by denar90**
#452

### CODE REVIEW

**Comment by jdaudier** [comment]
@denar90 We don't need to review this yet since it's WIP, correct?

**Comment by denar90** [comment]
Can you do it partly or it's easy just do review of all final version?

**Comment by peterblazejewicz** [comment]
> Can you do it partly or it's easy just do review of all final version?

Yes, the PR can be run locally. Thanks!

**Comment by jdaudier** [comment]
I prefer to review the final, so just ping me whenever you are ready @denar90. No rush, thanks.

**Comment by denar90** [comment]
got it)

**Comment by denar90** [comment]
I've finished.
But maybe you will have some comments about how it looks like or about cut sections from main page.
If `jade` sections is no longer needed in main page I'll remove them (didn't do it because they might be back).
Also have question about youtube api key. Should it be in [here](https://github.com/denar90/marionettejs.com/blob/ar-page/tasks/consts.js#L18) or moved somewhere?

**Comment by jdaudier** [comment]
@denar90 Do you mind fixing the conflict?

**Comment by denar90** [comment]
@jdaudier done)

File: src/additional-resources/sections/_header.jade [hunk 1]
```diff
@@ -7,6 +7,6 @@ include ../../sections/_nav
       include ../../partials/_marionette_logo
       include ../../partials/_marionette_text
     .right
-      h1 Latest updates
-      p Check here for new versions and commit logs in Marionette.
+      h1(itemprop="alternativeHeadline") Additional Marionette Resources
+      p(itemprop="description") Now learn Marionette easy than ever
```
Comment (scott-w) [inline src/additional-resources/sections/_header.jade]: `Marionette is now easier to learn than ever` would read better

**Comment by scott-w** [comment]
👍

**Comment by denar90** [comment]
@jdaudier here some screenshots how it looks like
Main page became simpler, some sections were removed, now they are present on AR page.
https://monosnap.com/file/gh69dtO9rpAVuseTSA4KcapKRYZ17e

Some screens of AR page
https://monosnap.com/file/8tbzregQAi1oPPoPHHKTGsR8WPZnbA
https://monosnap.com/file/Krc7WRr9798S1e9EC8Dk8A6mCCpLDn
https://monosnap.com/file/qBRTgLYpTb7jAAXo4WEzjo6ucVYPU1
https://monosnap.com/file/9w8ze8xfuooCwiwheuF6EPuPjq7SqO

**Comment by denar90** [comment]
Here also some video https://monosnap.com/file/66E7JhY2DWM5bRdXiRSkdsPgeKIgqw

**Comment by jdaudier** [comment]
Thanks @denar90 for the awesome work!

**Pull request merged**